src/api/app/controllers/webui/cloud/ec2/configurations_controller.rb

```diff
@@ -5,7 +5,6 @@ class ConfigurationsController < WebuiController
         before_action :require_login
 
         def show
-          switch_to_webui2
           @ec2_configuration = User.session!.ec2_configuration || User.session!.create_ec2_configuration
           @aws_account_id = CONFIG['aws_account_id']
```

Comment:
Those two lines can go right?

Reply:
No, they are still executed. They would need to be removed if it would have been `return if switch_to_webui2`.